packages/kit/types/ambient.d.ts

```diff
@@ -119,16 +119,18 @@ declare module '$app/stores' {
 		session: Writable<App.Session>;
 		updated: typeof updated;
 	};
-	/**
-	 * A readable store whose value contains page data.
-	 */
-	export const page: Readable<{
+
+	interface Page<Params extends Record<string, string> = Record<string, string>> {
```

Comment:
if this is going to close https://github.com/sveltejs/kit/issues/3730 then I think this interface would need to be exported?